templates/mediawiki-apb.yaml

```diff
@@ -3,7 +3,7 @@ kind: ServiceInstance
 metadata:
   name: mediawiki123
```

Comment:
did you want to change this?

Reply:
No, check the collapsed conversation :)

Reply:
> There's a limitation in CI where the deployment/deploymentconfig needs to have the name as the service instance resource so bind creds get delivered to the right place.
> This needs to be mediawiki:
https://github.com/ansibleplaybookbundle/mediawiki-apb/blob/master/roles/provision-mediawiki123-apb/tasks/main.yml#L26
>
If we change it then then the resource name in the playbook needs to change too. We decided to keep it so we don't make a playbook change.

Reply:
lol yuck. Makes sense to me. I'll forget tomorrow though.